lib/caesar.rb

```diff
@@ -1,6 +1,57 @@
 class Caesar
-  attr_reader :key
 
-  def initialize
+  def eng_to_cipher(phrase, count)
+    new_phrase = phrase.chars.map do |letter|
+      if letter == letter.upcase
+        upper_case_transform(letter, count)
+      else 
+        lower_case_transform(letter, count)
+      end
+    end
+    puts new_phrase.join
   end
-end
\ No newline at end of file
+
+  def upper_case_transform(letter, count)
+    letters = ("A".."Z").to_a.join
+    new_letters = letters.chars.rotate(-count).join
+    new_letter = letter.tr(letters, new_letters)
+  end
+
+  def lower_case_transform(letter, count)
```

Comment:
This is so close to upper case transform, id probably use a single method perhaps have it decide which alphabet to use.